android/app/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Mon Jan 27 17:42:41 EET 2020
```

Comment:
what is this file?

Reply:
I don't know. it was generated automatically. I will try to remove and rebuild